rewrite.go

```diff
@@ -68,12 +68,13 @@ func rewriteGoFile(name, qual string, paths []string) error {
 
 	var changed bool
 	for _, s := range f.Imports {
-		name, err := strconv.Unquote(s.Path.Value)
+		importName, err := strconv.Unquote(s.Path.Value)
 		if err != nil {
 			return err // can't happen
 		}
-		q := qualify(unqualify(name), qual, paths)
-		if q != name {
+		q := qualify(unqualify(importName), qual, paths)
+		if q != importName {
+			printVerbose("rewrite "+name, s.Path.Value, "→", q)
```

Comment:
The → looks nice, but it can be difficult to process with other tools like sed and grep. Let's try to stick to ASCII.